tasklist/client/src/common/tasks/details/TaskDetailsHeader/index.tsx

```diff
@@ -43,7 +44,7 @@ const TaskDetailsHeader: React.FC<Props> = ({
         <span className={styles.processName}>{processName}</span>
       </div>
       <div className={taskDetailsLayoutCommon.headerRightContainer}>
-        {taskState === 'COMPLETED' ? (
+        {taskState === 'COMPLETED' && (
```

Comment:
@vitorwtavares We have a lib called `ts-pattern` that we can use for pattern matching here
I believe it will make this cleaner since it has a lot of cases

Please check on the codebase examples of how it's used